src/wp-includes/pluggable.php

```diff
@@ -2103,6 +2122,7 @@ function wp_new_user_notification( $user_id, $deprecated = null, $notify = '' )
 			'subject' => __( '[%s] Login Details' ),
 			'message' => $message,
 			'headers' => '',
+			'send'    => true,
```

Comment:
I'm unclear what this is for, where is it used in `wp_mail()`?

:1234: This applies to both set of arguments: for user and admin.